xpdacq/beamtime.py

```diff
@@ -209,8 +212,9 @@ def register_scanplan(self, scanplan):
         if scanplan.short_summary() not in sp_name_list:
             self.scanplans.append(scanplan)
         else:
-            old_obj = [ obj for obj in self.scanplans
-                      if obj.short_summary() ==scanplan.short_summary()].pop()
+            old_obj = [obj for obj in self.scanplans
+                       if
+                       obj.short_summary() == scanplan.short_summary()].pop()
             old_obj_ind = self.scanplans.index(old_obj)
```

Comment:
Do we need to separate conditional statement every time? I haven't noticed a strict style in PEP 8